python/tchannel/sync/client.py

```diff
@@ -25,8 +25,8 @@
 from threadloop import ThreadLoop
 from tornado import gen
 
-from . import glossary
-from . import tornado as async
+from tchannel import glossary
+from tchannel import tornado as async
```

Comment:
My understanding is that you don't want to use relative imports to grab packages in parent modules; only children.